src/web/mjs/connectors/templates/MH.mjs

```diff
@@ -0,0 +1,69 @@
+import Connector from '../../engine/Connector.mjs';
+import Manga from '../../engine/Manga.mjs';
+
+export default class MH extends Connector {
+
+    constructor() {
+        super();
+        super.id = undefined;
+        super.label = undefined;
+        this.tags = [];
+        this.url = undefined;
+
+        this.path = '/booklist?page=';
+        this.pathMatch = 'page=(\\d+)';
+        this.queryMangasPageCount = 'div.page-pagination div.pagination li:nth-last-child(2) a';
+        this.queryMangas = 'ul.mh-list li div.mh-item-detali h2.title a';
+        this.queryMangasURL = 'section.banner_detail div.info h1';
```

Comment:
Variable name implies to be used in `_getMangas`, but in fact this is used to extract a title...
May rename to something like `queryMangaTitle`